### PR TITLE
Added Symfony 3 support

### PR DESCRIPTION
**Pull request by gnugat**
Hi everyone,

Here's a small tweak to make this bundle compatible with Symfony 3. I've run the tests and checked it on a demo application and everything seems fine.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,15 +12,15 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "symfony/symfony"                : "~2.7",
-        "react/react"                    : "~0.4"
+        "php": "^5.5|^7.0",
```
Comment (mikaelrandy) [inline composer.json:15]: +:100:

**Comment by mikaelrandy** [comment]
:+1:

**Comment by omansour** [comment]
:+1:

**Comment by fabdsp** [comment]
:+1:

**Comment by fabdsp** [comment]
v1.1.1

**Pull request merged**